src/Microsoft.Diagnostics.EventFlow.Outputs.ElasticSearch/Configuration/ElasticSearchProxyConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.EventFlow.Configuration
+{
+    public class ElasticSearchProxyConfiguration
+    {
+        public string Uri { get; set; }
```

Comment:
Is there a reason not to use type of `Uri` or `Uri?` directly?

Reply:
It is a string in order to avoid dealing with malformed URIs at deserialization time.

Reply:
I see. That makes sense.